lib/inspec/resources/docker_container.rb

```diff
@@ -43,6 +43,14 @@ def running?
       status.downcase.start_with?("up") if object_info.entries.length == 1
     end
 
+    def has_volume?(destination, source)
+      # Todo:
+      # [+] Raise proper exception for containers with no mounted volume instead of saying undefined method `Destination' for nil:NilClass
+      raise Inspec::Exceptions::ResourceFailed, "Could not find any mounted volumes for your container" unless volume_info.Mounts[0]
+
+      volume_info.Mounts[0].Destination == destination && volume_info.Mounts[0].Source == source
```

Comment:
What if the container has multiple mounts, and the queried pair is simply the second one listed? This would appear to return false in that case. I think you want 
```
volume_info.Mounts.detect { |mount| mount.Destination == destination && mount.Source == source }
```

Reply:
💯 correct.